resources/istio/files/istio-operator-cluster-evaluation.yaml

```diff
@@ -172,7 +172,7 @@ spec:
             maxSurge: 100%
             maxUnavailable: 25%
   meshConfig:
-    accessLogFile: /dev/stdout
+    accessLogFile: ""
```

Comment:
I can add docs about it in comment, what do u think?

Reply:
That's a good idea. Please add it in every file, so nobody forgets about it :)